### PR TITLE
Clean up ignition-tools.pc file

### PR DESCRIPTION
**Pull request by scpeters**
# 🦟 Bug fix

Fixes minor problem introduced by #30 

## Summary

We merged #30 to make our cmake and `.pc` config files relocatable. That PR was merged before we noticed an issue in a similar PR: https://github.com/ignitionrobotics/ign-cmake/pull/129. The problem is the hard-coded `../..` path to prefix from `pcfiledir` is incorrect when the `.pc` file is installed to a subfolder of lib, such as for Ubuntu debian packages on amd64, which install to `/usr/lib/x86_64-linux-gnu/pkgconfig`. This PR includes the fix that was used for ign-cmake.

I also noticed that we have include and linker flags for this package in the `.pc` file even though it has no libraries or header files. So I've removed those files as well. Of course, now the `.pc` file is  basically a no-op, but I'd rather have the correct relocatable path logic even if it's not used.

## Checklist
- [X] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: cmake/pkgconfig/ignition.in [hunk 1]
```diff
@@ -1,10 +1,10 @@
-prefix=${pcfiledir}/../..
-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+prefix=${pcfiledir}/@PC_CONFIG_RELATIVE_PATH_TO_PREFIX@
+libdir=${prefix}/@LIB_INSTALL_DIR@
 includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
```
Comment (scpeters) [inline cmake/pkgconfig/ignition.in:3]: we could just delete these 3 lines I suppose

Comment (j-rivero) [reply cmake/pkgconfig/ignition.in:3]: +1, maybe we can just drop a comment about the lack of headers/libs so anyone looking into the generated file don't panic about the lack of data.

File: cmake/pkgconfig/ignition.in [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 Name: Ignition @IGN_PROJECT_NAME@
 Description: A set of @IGN_PROJECT_NAME@ classes for robot applications
 Version: @PROJECT_VERSION_FULL@
 Requires:
-Libs: -L${libdir} -l@PROJECT_NAME_LOWER@
-CFlags: -I${includedir}
+Libs:
```
Comment (j-rivero) [inline cmake/pkgconfig/ignition.in:9]: Looking into the [.pc file format](http://manpages.ubuntu.com/manpages/focal/man5/pc.5.html) seems like we can remove the empty fields. And that somehow there is an URL field mandatory :roll_eyes:

Comment (scpeters) [reply cmake/pkgconfig/ignition.in]: empty fields removed in c20c209d960742e0c71c918cd636de8a09caeb37

continuing to ignore the URL field

File: cmake/pkgconfig/ignition.in (hunk 1 above)
Comment (scpeters) [reply cmake/pkgconfig/ignition.in]: comment added in 2a90e1701b193f79a5ca17e8f88ab345b985dcbc, unneeded lines removed in c20c209d960742e0c71c918cd636de8a09caeb37

**Review by j-rivero — APPROVED** [review]

**Comment by j-rivero** [comment]
A side comment: having a `.pc` with no information about compilation is quite weird. A signal that we want rethink somehow what are doing with it and how because there are probably better alternatives.

**Comment by scpeters** [comment]
> A side comment: having a `.pc` with no information about compilation is quite weird. A signal that we want rethink somehow what are doing with it and how because there are probably better alternatives.

other packages may list it as `Required`, in which case we need a file for them to find, even if it is empty

**Pull request merged**